src/features/pass-survey/model/hooks/useActivityStepper.ts

```diff
@@ -48,6 +51,29 @@ function useActivityStepper(state: ActivityState | undefined) {
     return valid;
   }
 
+  function getNextStepShift(direction: 'forwards' | 'backwards') {
+    let shift = 1;
+
+    while (true) {
+      let nextStep = direction === 'forwards' ? step + shift : step - shift;
+
+      if (nextStep > items.length - 1) {
+        shift = items.length;
+        break;
+      }
+
```

Comment:
Can nextStep be less than 0? If so, shoud "break" be called?

Reply:
It cannot be less than zero because according to requirements, it is not possible that we can define a condition that depends on the next items. But i will handle this just in case